packages/pluggableWidgets/image-viewer-web/src/ImageViewer.editorConfig.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    hidePropertiesIn,
+    Properties,
+    StructurePreviewProps,
+    transformGroupsIntoTabs
+} from "@mendix/piw-utils-internal";
+import { DatasourceEnum, ImageViewerPreviewProps } from "../typings/ImageViewerProps";
+
+type ImageViewPreviewPropsKey = keyof ImageViewerPreviewProps;
```

Comment:
`keyof` just creates an enum and if I create a variable and use this type on it, it would also be a singular type. For me it makes sense to keep this singular. What is the reason you think this should be plural?